views/default/admin_module_basic.pdt

```diff
@@ -85,6 +94,28 @@
                     </tbody>
                 </table>
             </div>
+
+            <div id="static_tlds_section">
+                <div class="title_row"><h3><?php $this->_('AdminModule.basic.heading_tlds');?></h3></div>
+                <div class="pad">
+                    <ul>
+                        <li>
+                            <?php
+                            $this->Form->fieldCheckbox('static_tlds', '1', (isset($vars['static_tlds']) ? $vars['static_tlds'] : '1') == '1', ['class' => 'inline', 'id' => 'static_tlds']);
+                            $this->Form->label($this->_('AdminModule.basic.static_tlds', true), 'static_tlds', ['class' => 'inline']);
+                            ?>
+                            <span class="tooltip block"><?php $this->_('AppController.tooltip.text');?><div><?php $this->_('AdminModule.basic.tooltip_static_tlds');?></div></span>
+                        </li>
+                        <li id="static_tlds_fields">
+                            <?php
+                            $this->Form->label($this->_('AdminModule.basic.tlds', true), 'tlds');
+                            $this->Form->fieldText('tlds', (isset($vars['tlds']) ? $vars['tlds'] : null), ['id' => 'tlds', 'placeholder' => $this->_('AdminModule.basic.placeholder_tlds', true)]);
```

Comment:
Looks like 'AdminModule.basic.placeholder_tlds' didn't get defined in the language file